js/src/Behavior.js

```diff
@@ -399,6 +404,27 @@ function toggle_label_mouseover (on_off) {
 }
 
 /**
+ * With no argument, toggle the tooltips upon touching of labels.
+ * @param {Boolean} on_off - The new on/off state.
+ */
+function toggle_label_touch (on_off) {
+  if (on_off === undefined) {
+    on_off = this.label_touch === null
+  }
+
+  if (on_off) {
+
+    this.label_touch = function (type, d) {
```

Comment:
No need to use `.bind(this)` if using ES6 arrow function:
```
    this.label_touch = (type, d) => {
      if (!this.dragging) {
         this.map.callback_manager.run('show_tooltip', null, type, d)
      }
    }
```

Reply:
Done